st2actions/st2actions/runners/ssh/parallel_ssh.py

```diff
@@ -242,7 +242,7 @@ def _run_command(self, host, cmd, results, timeout=None):
             results[host] = {'stdout': stdout, 'stderr': stderr, 'return_code': exit_code,
                              'succeeded': is_succeeded, 'failed': not is_succeeded}
         except:
-            error = 'Failed executing command %s on host %s', cmd, host
+            error = 'Failed executing command %s on host %s' % (cmd, host)
```

Comment:
Also, I assume that this resulted in an exception which was caught in the runner container, right?